welltestpy/data/testslib.py

```diff
@@ -467,6 +467,26 @@ def plot(self, wells, exclude=None, fig=None, ax=None, **kwargs):
             **kwargs
         )
 
+    def diagnostic_plot(self, wells, **kwargs):
+        """Make a diagnostic plot.
+
+
+        Parameters
+        ----------
+       wells : :class:`dict`
```

Comment:
Please rename `wells` to `observation_well` to be in line with the routine in campaign.